src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input {
+  padding: 23px 62px;
+  border: none;
+  outline: none;
+  font-size: 16px;
+  background: #fff;
+  width: 100%;
+  height: 100%;
+  line-height: 22px;
+  font-family: Avenir, sans-serif;
```

Comment:
move this property to `html` selector & use `inherit` where needed

Reply:
done